Chatkit/Redux/Store/State.swift

```diff
@@ -0,0 +1,34 @@
+
+enum Internal {}
+
+extension Internal {
+    struct Room {
+        let identifier: String
+        let name: String
+    }
+}
+
+extension Internal.Room: Equatable {}
+
+extension Internal {
+    struct User {
+        let identifier: String
+        let name: String
+    }
+}
+
+extension Internal.User: Equatable {}
+
+struct State {
+    let currentUser: Internal.User?
+    let joinedRooms: [Internal.Room]
+    
+    static var emptyState: State {
```

Comment:
It makes sense in my opinion to explicitly mark this as `let`.

It would also feel a bit more "Swifty" when named `empty` (`State.empty` instead of `State.emptyState`).

Reply:
Both good suggestions 👍 .  Will change.

Reply:
This now done FYI